README.md

```diff
@@ -0,0 +1,17 @@
+Currently, this project reads the warnings from Checker Framework and calls Specimin on the right Field or Method.
+
+Usage:
+
+./gradlew run -PappArgs="project-root warning-log-file CFWR-root"
```

Comment:
When specifying a command line like this one that includes variables that the user is supposed to fill in, be explicit about the fact that you are doing so. There are two parts to this:
* add text directly above or below the command line that briefly explains each argument
* use some kind of syntax that sets off variables from text that should be copied directly

For the latter, I usually use bash syntax, like this:
```
project-root=...
warning-log-file=...
CFWR-root=...
./gradlew run -PappArgs="${project-root} ${warning-log-file} ${CFWR-root}"
```

Reply:
(if you don't do this, users will directly copy-paste the command line and then complain to you that it does not work)